### PR TITLE
feat(publication-date): add story, css, tokens

### LINKED ISSUE #652: Design: Publication date
https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Design-System---Gemeente-Den-Haag?node-id=8515%3A28504

### PR DESCRIPTION
**Pull request by robbieatacato**
## Solving:
https://github.com/nl-design-system/denhaag/issues/652

### Publication date + edited date:
- html
- responsive css
- design tokens

closes #652

### CODE REVIEW

File: components/PublicationDate/src/stories/publicationDate.bem.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
```
Comment (bddjong) [inline components/PublicationDate/src/stories/publicationDate.bem.stories.mdx:1]: This file should probably be called `bem.stories.mdx`. We don't use the component name in the stories files anymore.

**Review by bddjong — CHANGES_REQUESTED** [review]

File: components/PublicationDate/src/stories/publicationDate.bem.stories.mdx (hunk 1 above)
Comment (robbieatacato) [reply components/PublicationDate/src/stories/publicationDate.bem.stories.mdx]: @bddjong I renamed the story file in commit: https://github.com/nl-design-system/denhaag/pull/662/commits/ed93c2636a22338d5b7acaf1c5e49d3d4e01f4c3

**Review by bddjong — APPROVED** [review]

File: proprietary/Components/src/denhaag/publication-date.tokens.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "denhaag": {
+    "publication-date": {
+      "border": { "value": "1px solid #D2D2D2" },
+      "color": { "value": "{denhaag.color.grey.4}" },
+      "font-family": { "value": "{denhaag.typography.sans-serif.font-family}" },
+      "font-weight": { "value": "400" },
+      "font-size": { "value": "1.125rem" },
```
Comment (bddjong) [inline proprietary/Components/src/denhaag/publication-date.tokens.json:8]: I noticed this font size is not defined in the typography tokens. We only have `1rem` and `1.125rem`. Are our typography tokens outdated? Or is this just an exception.

**Review by bddjong — APPROVED** [review]

File: proprietary/Components/src/denhaag/publication-date.tokens.json (hunk 2 above)
Comment (robbieatacato) [reply proprietary/Components/src/denhaag/publication-date.tokens.json:8]: @bddjong this is an exception that follows the UX documentation in figma

**Pull request merged**